### DISCUSSION TITLE
log4j vulnerability

### DISCUSSION THREAD

**Discussion by beechwood13** [GMOD/jbrowse#1621] [General]
Hi, It appears the JBrowse has log4j security issue:

===============================================================================

[WARNING] [42 - contains log4j files] /usr/local/jbrowse/node_modules/dojo-util/buildscripts/cldr/lib/bsf.jar
[WARNING] [43 - contains log4j files] /usr/local/jbrowse/node_modules/dojo-util/buildscripts/cldr/lib/commons-logging-1.1.1.jar
[WARNING] [55 - contains log4j files] /usr/local/jbrowse-1.16.10-release/node_modules/dojo-util/buildscripts/cldr/lib/bsf.jar
[WARNING] [56 - contains log4j files] /usr/local/jbrowse-1.16.10-release/node_modules/dojo-util/buildscripts/cldr/lib/commons-logging-1.1.1.jar
[WARNING] This script does not guarantee that you are not vulnerable, but is a strong hint.
============================================================================== 

I am wondering whether a security update for JBrowse 1.x is planned in the near future ?

**Comment by cmdcolin** [comment]
Hi there

Similar post here
https://sourceforge.net/p/gmod/mailman/message/37411597/

Just as an added note suggested by @scottcain you could delete those files and the code would likely still work
